packages/ocamlformat/ocamlformat.0.13.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "OCamlFormat Team <ocamlformat-team@fb.com>"
+authors: "Josh Berdine <jjb@fb.com>"
+homepage: "https://github.com/ocaml-ppx/ocamlformat"
+bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
+dev-repo: "git+https://github.com/ocaml-ppx/ocamlformat.git"
+url {
+  src:
+    "https://github.com/ocaml-ppx/ocamlformat/releases/download/0.13.0/ocamlformat-0.13.0.tbz"
+  checksum: [
+    "sha256=3f7bc85bb26284b49a818a6ea4e13919e709a24cded67f674f1aa61347703464"
+    "sha512=3939bb3b6103e020b046c7ec7ff7134e4ea37f27f694bb426c30bbf6574eb4daeb58ffd5631331680512fb68ed425c34ab2f5c54d38b808960e8c95b7af80530"
+  ]
+}
+license: "MIT"
+build: [
+  ["ocaml" "tools/gen_version.mlt" "lib/Version.ml" version] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test & ocaml:version >= "4.08"}
```

Comment:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
Since OCaml >= 4.08 is already required this is not necessary anymore.
However tests are failing anyway, could you have a look at the logs?
* https://ci.ocaml.org/log/saved/docker-run-96c128a9e7b0fcb3431aca97fe96f8c1/09bacd487596bb99849cc645048335c521c3a300
* https://ci.ocaml.org/log/saved/docker-run-6525eec698fe84be16d761665b40ecbe/ffb15ee575d70d7e4dc0671c21dc8d2cd66e4221
* https://ci.ocaml.org/log/saved/docker-run-4eb93e2ff7a92bc663867f9ae731eb91/02e3bbacfd08d4aa7500ca2119a4bd9265e6851e